CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ changes.
 
 - **DO NOT RELEASE** as only tested against Sanchonet.
```

Comment:
Can we test this again on preview and remove this then?

Reply:
Yes, after this.